.NET/Microsoft.Recognizers.Text.DataDrivenTests/DateTime/TestDateTime_Hindi.cs

```diff
@@ -0,0 +1,199 @@
+﻿using System.Collections.Generic;
+using Microsoft.Recognizers.Text.DataDrivenTests;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.Recognizers.Text.DateTime.Tests
+{
+    [TestClass]
+    public class TestDateTime_Hindi : TestBase
+    {
+        public static IDictionary<string, IDateTimeExtractor> Extractors { get; private set; }
+
+        public static IDictionary<string, IDateTimeParser> Parsers { get; private set; }
+
+        [ClassInitialize]
+        public static void ClassInitialize(TestContext context)
+        {
+            Extractors = new Dictionary<string, IDateTimeExtractor>();
+            Parsers = new Dictionary<string, IDateTimeParser>();
+        }
+
+        [NetCoreTestDataSource]
+        [TestMethod]
+        public void DateExtractor(TestModel testSpec)
+        {
+            ExtractorInitialize(Extractors);
+            TestDateTimeExtractor(testSpec);
+        }
+
+        ////[NetCoreTestDataSource]
```

Comment:
All these test calls can be uncommented, as long as the specs are properly marked as NotSupported during dev.

Reply:
we will uncomment with the PRs for each of the subtypes (that include the specs)

Reply:
@LionbridgeCS please uncomment all at once and add the spec files even if empty or with only one disabled case per file. 
The recommendation is to always add either all new language specs marked as NotSupported, or to add the skeleton of specs and test code for the whole type.
There could be a simple PR just for this. The main goal here is to show that all skeleton code runs and executes it's tests and there is no infra-structure failure.

Reply:
@tellarin: noted. we will make a simple PR with this.